lib/node_modules/@stdlib/array/empty-like/docs/types/index.d.ts

```diff
@@ -475,8 +475,8 @@ declare function emptyLike( x: Complex64Array, dtype?: DataType ): Complex64Arra
 * @example
 * var zeros = require( `@stdlib/array/zeros` );
 *
-* var x = zeros( 2, 'float32' );
-* // returns <Float32Array>[ 0.0, 0.0 ]
+* var x = zeros( 2, 'int32' );
+* // returns <Int32Array>[ 0.0, 0.0 ]
```

Comment:
```suggestion
* // returns <Int32Array>[ 0, 0 ]
```

For integer arrays, we can omit the decimal points.

@Planeshifter This may be an opportunity for a lint rule. Namely, if an integer typed array, the buffer values should not include decimal points.

@steff456 Would you mind creating an internal todo for creating the lint rule?